### ISSUE TITLE
warmup epochs

### ISSUE THREAD

**Issue by mohamedabdallah1996** [experiencor/keras-yolo2#293]
please any one can clarify:
1. what is the warmup epochs ?
2. when should I use it ?

thanks

**Comment by sugi-chan** [comment]
Hi, 

As I understand for the projects I have done using this repository I use "warmup" initially to let a model adjust to a new dataset. It gets talked about in some other [issues](https://github.com/experiencor/keras-yolo2/issues/164). Basic pipeline has 2 steps, first is to run training with warmup for 3-5 epochs initially to let it adjust (you can save those weights), then once it finishes you can use those weights to do standard training. Hope that helps a bit.
